suite-native/module-settings/src/components/CryptoUnitsSelector.tsx

```diff
@@ -25,7 +25,7 @@ export const CryptoUnitsSelector = () => {
 
     return (
         <Select<PROTO.AmountUnit>
-            selectLabel="Bitcoin Amount Units"
+            selectLabel="Bitcoin units"
```

Comment:
I know this is not part of the task, but since you are touching this, what do you think about adding intl (change selectLabel to ReactNode and use ) here, so we continue gradually adding translations everywhere.

Reply:
Yep, I thought about it as well but decided not to change it now. I'm not sure ReactNode is a way to go, would prefer translation ID instead. And that's probably something that might deserve a team discussion/consensus first.

Reply:
💪  I am fine with translationID as well, since this is only used twice, it wont make any difference.

Reply:
We already use ReactNode is most of places so I would use that also here. Passing only ID is problematic when you have translation with variables because you need to add also props for these variables and then typing become issue etc etc.

Reply:
`<Translation ... />` used.